src/azure-cli/azure/cli/command_modules/reservations/_help.py

```diff
@@ -162,3 +162,84 @@
     type: string
     short-summary: Id of the subscription to look up applied reservations
 """
+
+helps['reservations reservation-order calculate'] = """
+type: command
+short-summary: calculate price
+long-summary: calculate price for reservation
+parameters:
+  - name: --applied-scope-type
+    type: string
+    short-summary: choose Single or Shared as appliedscope
+  - name: --applied-scopes
+    type: string
+    short-summary: Subscription that the benefit will be applied. Do not specify if --applied-scope-type is Shared.
+  - name: --billing-plan
+    type: string
+    short-summary: choose Monthly or Upfront
```

Comment:
The same as https://github.com/Azure/azure-cli/pull/11242/files#r347789648

Reply:
I haven't seen it shows up as enum value in your command. Can you double check if it is resolved?